spec/jobs/warm_up_facility_group_analytics_cache_job_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe WarmUpFacilityGroupAnalyticsCacheJob, type: :job do
+  include ActiveJob::TestHelper
+
+  describe '#perform_later' do
+    let(:facility_group) { create(:facility_group) }
+    let!(:facilities) { create_list(:facility, 2, facility_group: facility_group) }
+
+    let(:from_time) { Date.new(2019, 1, 1).strftime('%Y-%m-%d') }
+    let(:to_time) { Date.new(2019, 3, 31).strftime('%Y-%m-%d') }
+
+    let(:job) { WarmUpFacilityGroupAnalyticsCacheJob.perform_later(facility_group, from_time, to_time) }
+
+    it 'queues the job' do
+      expect { job }.to change(ActiveJob::Base.queue_adapter.enqueued_jobs, :size).by(1)
```

Comment:
This helper can be used here instead: https://apidock.com/rails/v4.2.7/ActiveJob/TestHelper/assert_enqueued_jobs